app/javascript/.eslintrc.js

```diff
@@ -54,6 +54,7 @@ module.exports = {
     ],
     'react/jsx-no-target-blank': [2, { enforceDynamicLinks: 'always' }],
     'jsx-a11y/no-onchange': 'off',
+    'prefer-const': ['error'],
```

Comment:
I put the rule on the `app/javascript` eslint configuration instead of the root esint configuration because non ES module code uses `var` still in a lot of places.